lib/datasource/metadata.ts

```diff
@@ -117,6 +118,43 @@ function massageGitlabUrl(url: string): string {
     .replace('.git', '');
 }
 
+function normalizeDate(input: any): string | null {
+  if (
+    typeof input === 'number' &&
+    !Number.isNaN(input) &&
+    input > 0 &&
+    input <= Date.now() + 24 * 60 * 60 * 1000
+  ) {
+    return new Date(input).toISOString();
+  }
+
+  if (typeof input === 'string') {
+    const luxonDate = DateTime.fromISO(input, { zone: 'UTC' });
```

Comment:
Using DateTime is to avoid converting the timestamp to the machine's time? If so then let's document that so nobody refactors it to use Date later. Or is there somehow a test for that?

Reply:
I've written the comment, though in my local machine some snapshot tests are also failing because of `2012-01-01` parsing mismatch.